File: python_t3_full/Clone_2434.py
def checksum(msg) :
	s = 0
	for i in range(0, len(msg), 2) :
		w = ord(msg [i]) + (ord(msg [i + 1]) < < 8)
		s = carry_around_add(s, w)
	return ~ s & 0xffff


def checksum(msg):
    s = 0
    pairs = [msg[i:i+2] for i in range(0, len(msg), 2)]
    for pair in pairs:
        w = sum(ord(char) << 8 * j for j, char in enumerate(pair))
        s = carry_around_add(s, w)
    return ~ s & 0xffff

 

File: c_t4_full/Clone_2434.c
int main () {
    int x;
    int sum = 0;
    for (x = 1; x <= 1000; x++)
        if (x % 3 == 0 || x % 5 == 0)
            sum += x;
    printf ("%d\n", sum);
    return 0;
}




int sum_values(int x, int sum) {
    if(x > 1000)
        return sum;
    if (x % 3 == 0 || x % 5 == 0)
        sum += x;
    return sum_values(x+1, sum);
    }

int main () {
    int sum = sum_values(1, 0);
    printf ("%d\n", sum);
    return 0;
}


